tests/modules/ebi-metagenomics/foldcomp/compress/main.nf

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env nextflow
+
+nextflow.enable.dsl = 2
+
+include { FOLDCOMP_COMPRESS } from '../../../../../modules/ebi-metagenomics/foldcomp/compress/main.nf'
+
+workflow test_foldcomp_compress {
```

Comment:
maybe rename this to `test_foldcomp_compress_dir` so it's obvious it's testing an entire directory as input?

Reply:
Valid point. Changed